addons/hr_contract/static/src/widgets/tooltip_warning_widget.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="hr_contract.ContractWarningTooltip" owl="1">
+        <span
+            class="fa fa-exclamation-triangle text-danger o_calendar_warning ms-3"
+            data-tooltip-template="hr_contract.CalendarMismatch"
+            data-tooltip-info='{
+                "text":"Calendar Mismatch : The employee&apos;s calendar does not match this contract&apos;s calendar. This could lead to unexpected behaviors."
```

Comment:
I don't think the text will be translated this way.
Instead add a getter in the Component and return the Object with the text translated